src/CommandingOfficers/AWBW/BH/Sturm.java

```diff
@@ -127,7 +127,10 @@ private MeteorParams findTarget(GameMap map)
     {
       AWBWValueFinders.CostValueFinder finder = new AWBWValueFinders.CostValueFinder();
       finder.maxDamage = power;
-      return MeteorParams.planMeteorOnEnemy(map, map, myCommander, 2, finder);
+      MeteorParams meteor = MeteorParams.planMeteorOnEnemy(map, map, myCommander, 2, finder);
+      if( meteor.target == null )
+        meteor.target = new XYCoord(0, 0);
+      return meteor;
```

Comment:
Could this logic be shuffled inside the `planMeteor...` calls so that you don't have to remember to manually reset it after planning?

Also does this happen? Is AWBW behavior different than the carts?

Reply:
Probably, though I dunno if it really makes sense to go into the function signature there.
As far as I can tell, yes. Testing is like pulling teeth, but it does work like that in the Move Planner.

Also, the AWBW meteor targeting is different in a bunch of ways, so I dunno why you're surprised.

Reply:
What do you mean, "go into the function signature"? Seems like it would be straightforward to either a) modify the `MeteorParams` constructor to replace `null` with `(0, 0)`, or b) change `planMeteor...` to default the target to `(0, 0)` instead of `null` at the outset, so it's already the right thing if it doesn't get replace.

> I dunno why you're surprised.

Ignorance has that effect on people.

Reply:
Oh, gotcha. the (0, 0) variant is only for the AWBW dudes. My brain is still adjusting to thinking of that as an independent AW game/category.